examples/tsp.rs

```diff
@@ -73,10 +70,7 @@ impl Conshdlr for SubtourConshdlr {
         let edges_in_lp_sol = self
             .vars
             .iter()
-            .filter(|v| {
-                let val = unsafe { ffi::SCIPgetSolVal(model.scip_ptr(), null_mut(), v.1.inner()) };
-                val > 0.5
-            })
+            .filter(|v| model.current_val(v.1) > 0.5)
```

Comment:
```suggestion
            .filter(|(_edge, v)| model.current_val(v.1) > 0.5)
```
Since it's called `v` I'd rather to destructure it get and attach it to the variable.

Reply:
Added destructuring